plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/dialogs/connection/ConnectionPageDriver.java

```diff
@@ -124,7 +124,7 @@ public void createSorterControl(Composite controlsGroup) {
         orderGroup.setLayoutData(new GridData(GridData.HORIZONTAL_ALIGN_END));
         orderGroup.setLayout(new RowLayout());
         //new Label(orderGroup, SWT.NONE).setImage(DBeaverIcons.getImage(UIIcon.SORT));
-        new Label(orderGroup, SWT.NONE).setText("Sort by: ");
+        new Label(orderGroup, SWT.NONE).setText(CoreMessages.driver_connection_sort_by + " ");

```

Comment:
You can use `UIUtils#createControlLabel`. After that, you can avoid specifying the `:` in the localized message.

Reply:
Seems UIUtils#createControlLabel is not compatible with RowLayout. It leads to error:
`class org.eclipse.swt.layout.GridData cannot be cast to class org.eclipse.swt.layout.RowData (org.eclipse.swt.layout.GridData and org.eclipse.swt.layout.RowData are in unnamed module of loader org.eclipse.osgi.internal.loader.EquinoxClassLoader @32fdec40)`

Reply:
I see.